test/visual-diff/visual-tests

```diff
@@ -69,6 +81,29 @@
    *     "amp-feature-two"
    *   ],
    *
+   *
+   *   // [optional, EXPERIMENTAL] Allow running "dirty" JavaScript code in this
+   *   // AMP page, and enable JavaScript execution on the Percy side.
+   *   // The code being executed *must* be inlined inside a <script> tag, not
+   *   // be an externally loaded script via a <script src="..."> tag.
+   *   // The code *WILL BE EXECUTED TWICE*! Once locally, before the DOM is
```

Comment:
In the past, double execution of JS code was the reason behind bugs like https://github.com/percy/percy-capybara/issues/51. I assume something has changed since then?

Reply:
By removing the AMP library from the page before uploading it to percy, we can avoid a lot of unexpected behaviour. Of course, the authors of those inlined scripts should still be aware of the possibility that things would happen unexpectedly, hence why I suggest using a guard here. I'd like to see how people use this feature before we refine it (insert generic quote about premature optimization)